app/views/users/new.html.erb

```diff
@@ -0,0 +1,70 @@
+<h1>Create New User</h1>
+
+<!---
+<form action="/users" method="POST" accept-charset="UTF-8">
+  <input type="hidden" name="authenticity_token" value="<%= form_authenticity_token %>">
+  <p>
+    <label for="username">Enter Username</label><br>
+    <input type="text" id="username" name="user[username]">
+  </p>
+  <p>
+    <label for="email">Enter Email</label><br>
+    <input type="email" id="email" name="user[email]">
+  </p>
+  <p>
+    <label for="password">Enter Password</label><br>
+    <input type="password" id="password" name="user[password]">
+  </p>
+  <p>
+    <input type="submit" name="submit" value="Create Account">
+  </p>
+</form>
+-->
+
+<%
+=begin %>
+<%= form_tag("/users", method: "POST") do %>
```

Comment:
`[Optional]` as you are using the helper methods here, you can use the helper path too instead fo `/users`.